src/Features/Core/Portable/DesignerAttribute/AbstractDesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -84,6 +79,35 @@ public DesignerAttributeDiscoveryService()
             if (!project.SupportsCompilation)
                 return;
 
+            // We're looking for "System.ComponentModel.DesignerCategoryAttribute", which is defined only in
+            // System.ComponentModel.Primitives.dll.  Avoid looking at any project that doesn't have a reference to that
```

Comment:
So in .NET Core that might be the case, but in .NET Framework I see it's in System...which mind you pretty much anything will reference. This will probably need a change or just outright removal until we figure out something else to do here.

Reply:
ugh.  ok, removed.